src/components/va-accordion/va-accordion.tsx

```diff
@@ -42,6 +49,20 @@ export class VaAccordion {
     }
 
     const prevAttr = clickedItem.getAttribute('open') === 'true' ? true : false;
+
+    if (!this.disableAnalytics) {
+      const detail = {
+        componentName: 'Accordion',
+        action: prevAttr ? "collapse" : "expand",
+        details: {
+          header: clickedItem.header,
+          subheader: clickedItem.subheader,
+          level: clickedItem.level,
+        },
+      };
+      this.componentLibraryAnalytics.emit(detail);
+    }
```

Comment:
I like that this takes advantage of the `accordionItemToggled` event rather than duplicating the necessary things (like `prevAttr` inside of the child item.